sdk-fetch

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
We've started to use `#!/usr/bin/env bash`, but I left this untouched from [bottlerocket-sdk/sdk-fetch](https://github.com/bottlerocket-os/bottlerocket-sdk/blob/develop/sdk-fetch). Should we just switch both?

Reply:
Switching to `#!/usr/bin/env bash` in [bottlerocket-sdk/sdk-fetch](https://github.com/bottlerocket-os/bottlerocket-sdk/blob/develop/sdk-fetch) seemed to cause issues. I'll keep this as is, at least for now.